### PR TITLE
Creating a data model

### PR DESCRIPTION
**Pull request by olaishola05**
# Creating a data model

- Build project schema.
  - Create and run the migration for users, posts, comments, and likes files.
  - Table created and column names match the ERD diagram.
  - Foreign keys should be included.
  - All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20220427191552_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :Title
+      t.string :Text
+      t.integer :Comments_Counters
+      t.integer :Likes_Counter
```
Comment (bimbolabuari) [inline db/migrate/20220427191552_create_posts.rb:8]: Kindly use `snake_case` as this is part of Ruby's best practice 👍 

image

File: db/migrate/20220427190212_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.string :Bio
```
Comment (bimbolabuari) [inline db/migrate/20220427190212_create_users.rb:6]: Kindly use `snake_case` as this is part of Ruby's best practice 👍 

```
      t.string : name
      t.string : photo
      t.string : bio
```

Please check for other occurrences 👍

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @netman5, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Create migration file for `users`, `posts` , `likes`, `comments` table

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @netman5,


## Wow, you did it 🎉
Brilliant
## Thank you for refactoring your table columns to snake case, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**